src/processing/resources/page.js

```diff
@@ -14,6 +13,7 @@ const BODY_CREATED_EVENT_SCRIPT =
                var script = document.currentScript || document.scripts[document.scripts.length - 1];
                script.parentNode.removeChild(script);
           </script>`;
```

Comment:
Just found out what that: Since it's multiline string it will include indentation, so, then injected it will have some really broken indentation. To reduce amount of bytes passed to client and because of aesthetical reason use tag for templates from [dedent](https://www.npmjs.com/package/dedent) package.